src/run.js

```diff
@@ -124,8 +125,20 @@ const processPage = ({
     // a second time.
     let fulfilledPromise = false;
 
+    const tracker = createTracker(page);
     const safeReject = error => {
       if (!fulfilledPromise) {
+        if (error.message.startsWith('Navigation Timeout Exceeded')) {
+          const urls = tracker.urls();
+          if (urls.length > 1) {
+            error.message += `\nFor one of the following urls: ${urls.join(
```

Comment:
That sentence is odd. It'd make sense if it started with `...for once of the`. 

I'm thinking we can be more direct and blunt. E.g.:
```javascript
if (url.length) {
  error.message += `Tracked URLs that have NOT finished: ${urls.join(',')}`
}
```

Reply:
Also, I thin the word should be `URLs` (not `urls`) since it's an acronym of Uniform Resource Locator.